docs/src/docs/asciidoc/user/config-derive.adoc

```diff
@@ -0,0 +1,301 @@
+----
+[cols=".^~a,^.^~d,.^~a"]
+|===
+|[source,xml]
+----
+<ehcache>
+  <cache alias="cache">
+    <key-type>Long.class</key-type>
+    <value-type>Object.class</value-type>
+    <heap>10</heap>
+  </cache>
+</ehcache>
+----
+|&rarr;
+|[source,xml]
+----
+<ehcache>
+  <cache alias="cache">
+    <key-type>Long.class</key-type>
+    <value-type>Object.class</value-type>
+    <heap>10</heap>
+    <resilience>
+      com.example.ThrowingResilienceStrategy
+    </resilience>
+  </cache>
+</ehcache>
+----
+|===
+
+.updating a service configuration (changing a clustered cache's consistency)
+[source,java,indent=0]
+----
+include::{sourcedir37}/clustered/client/src/test/java/org/ehcache/clustered/client/docs/ConfigurationDerivation.java[tag=updateService]
+----
+[cols=".^~a,^.^~d,.^~a"]
+|===
+|[source,xml]
+----
+<ehcache>
+  <service>
+    <tc:cluster>
+      <tc:connection url="terracotta://example.com/cachemanager"/>
+    </tc:cluster>
+  </service>
+
+  <cache alias="cache">
+    <resources>
+      <tc:clustered-dedicated unit="MB">50</tc:clustered-dedicated>
+    </resources>
+    <tc:clustered-store consistency="strong"/>
+  </cache>
+</ehcache>
+----
+|&rarr;
+|[source,xml]
+----
+<ehcache>
+  <service>
+    <tc:cluster>
+      <tc:connection url="terracotta://example.com/cachemanager"/>
+    </tc:cluster>
+  </service>
+
+  <cache alias="cache">
+    <resources>
+      <tc:clustered-dedicated unit="MB">50</tc:clustered-dedicated>
+    </resources>
+    <tc:clustered-store consistency="eventual"/>
+  </cache>
+</ehcache>
+----
+|===
+
+=== Removing a service
+Removing a service often involves removing both service creation and a service configuration instances since a service
+instance its configuration are usually strongly coupled:
+
+.removing a service
+[source,java,indent=0]
+----
+include::{sourcedir37}/clustered/client/src/test/java/org/ehcache/clustered/client/docs/ConfigurationDerivation.java[tag=removeService]
+----
+<1> From all cache configurations...
+<2> remove any existing `ClusteredStoreConfiguration` instances.
+<3> Create a new resource pool builder...
+<4> From the existing resource pools...
+<5> filter out any clustered resources.
+<6> Add all remaining pools to the new resource pools instance
+<8> Finally remove the clustering service creation configuration
+
+[cols=".^~a,^.^~d,.^~a"]
+|===
+|[source,xml]
+----
+<ehcache>
+  <service>
+    <tc:cluster>
+      <tc:connection url="terracotta://example.com/cachemanager"/>
+    </tc:cluster>
+  </service>
+
+  <cache alias="cache">
+    <resources>
+      <heap>100</heap>
+      <tc:clustered-dedicated unit="MB">50</tc:clustered-dedicated>
+    </resources>
+    <tc:clustered-store consistency="strong"/>
+  </cache>
+</ehcache>
+----
+|&rarr;
+|[source,xml]
+----
+<ehcache>
+  <cache alias="cache">
+    <resources>
+      <heap>100</heap>
+    </resources>
+  </cache>
+</ehcache>
+----
```

Comment:
The build output is complaining that the table marker (`|===`) is missing here.

Reply:
Fixed.